CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
-# v22.0.0-snapshot
+# v22.6.0a1
```

Comment:
Shall we also add the changelog and readme for v22b1 and v22ga? It's okay if you want to do them in separate PRs.